### DISCUSSION TITLE
language manager init_from_config question

### DISCUSSION THREAD

**Discussion by p0p4k** [coqui-ai/TTS#2468] [General Q&A] [answered]
Why do we not return language manager if the first `if` statement is satisfied? Cause then the nametoid dict inside will get reset in the baseidmanager. Is there a return missing?
https://github.com/coqui-ai/TTS/blob/c2d15cd41335798bfa81bc9376bdd35e5e187590/TTS/tts/utils/languages.py#L95-L98

**Comment by erogol** [comment, marked as answer]
I think its a bug 🐛
